LLama/Batched/Conversation.cs

```diff
@@ -206,6 +258,43 @@ private void AssertCanBePrompted()
 
         if (RequiresInference)
             throw new AlreadyPromptedConversationException();
+
+        _mtmdLogitsIndex = null;
+    }
+
+    public void QueueMedia(string path)
+    {
+        AssertCanBePrompted();
+
+        if (Executor.ClipModel is null)
+            throw new InvalidOperationException("This conversation is not configured for multimodal prompts.");
+
+        var embed = Executor.ClipModel.LoadMedia(path);
+        _mtmdEmbeds.Add(embed);
+        _mtmdLogitsIndex = null;
+    }
+
+    public void QueueMedia(SafeMtmdEmbed embed)
```

Comment:
Could we add an overload of Prompt that takes a `Span<SafeMtmdEmbed>` instead of enqueuing them? This removes some state and makes the ownership of the embed objects clearer (e.g. at the moment you could call `QueueMedia`, then dispose the media, which would trigger an error on the next call to `Prompt` when you tried to use that disposed object).